### ISSUE TITLE
Create table dynamically at runtime

### ISSUE THREAD

**Issue by ismailsimsek** [jdbi/jdbi#2058]
is it possible to create table dynamically without DAO object ?  something similar to  http://jdbi.org/#_updates

i get table structure in runtime fields + PK details and looking into a way to create the table if not exists (database agnostic way)

**Comment by ismailsimsek** [comment]
the issue is mapping the type field to vendor(database) type . String maps to VARCHAR but for oracle it should map to VARCHAR2.

example:
```java
create.createTable("mytesttable")
          .column("id", Types.BIGINT)
          .column("coll1", Types.LONGNVARCHAR)
          .column("coll2", Types.NUMERIC)
          .column("coll3", Types.FLOAT)
          .column("coll4", Types.DECIMAL)
          .primaryKey("id"))
        .create()
```

**Comment by stevenschlansker** [comment]
Yes, this is possible to do, but Jdbi does not provide automatic conversion of DDL statements between different SQL dialects. We consider this out of scope for our project since it is a very large and complicated feature to implement. You would either provide schema migrations written for each database you support individually, or alternatively might look into another library like [jOOQ](https://www.jooq.org) which can be used in addition to (or instead of) jdbi to accomplish what you like.
